protocols/v2/subprotocols/mining/src/update_channel.rs

```diff
@@ -6,41 +6,49 @@ use binary_sv2::{Deserialize, Serialize, Str0255, U256};
 #[cfg(not(feature = "with_serde"))]
 use core::convert::TryInto;
 
-/// # UpdateChannel (Client -> Server)
+/// Message used by downstream to notify an upstream about changes on a specified channel.
 ///
-/// Client notifies the server about changes on the specified channel. If a client performs
-/// device/connection aggregation (i.e. it is a proxy), it MUST send this message when downstream
-/// channels change. This update can be debounced so that it is not sent more often than once in a
-/// second (for a very busy proxy).
+/// If a downstream performs device/connection aggregation (i.e. it is a proxy), it must send this
+/// message when downstream channels change.
 ///
-/// This message is an extended channel only message. Using it in other kind if channels should
-/// raise an error
+/// Only relevant for Extended Channels.
```

Comment:
where did this information come from?

Reply:
see the removed line just above it, it starts with `This message is an Extended Channel only message`